### ISSUE TITLE
OSX Ledger Nano S errors

### ISSUE THREAD

**Issue by dgobaud** [spesmilo/electrum#3957]
these screenshots are from Electrum-LTC but the errors messages/process in Electrum is the same.

In general have many problems using the Nano with Electrum. I figured out how to get it to work but very annoying.

Open ledger bitcoin app on device with browser support off.

Start creating a wallet choose Standard wallet then Use a hardware device. It finds the ledger and gives "read error"

![image](https://user-images.githubusercontent.com/240811/36554594-b39d51ac-17cd-11e8-9b9b-cf8b74eb4408.png)

now turn browser support on and click next again

![image](https://user-images.githubusercontent.com/240811/36554665-e5f96172-17cd-11e8-8ef1-a981b9779f13.png)

now turn browser support off and click next sometimes it works or sometimes get another error then click back then go forward again and it works.

basically cycling browsersupport and constantly trying seems to make it work eventually

**Comment by SomberNight** [comment]
dupe of https://github.com/spesmilo/electrum/issues/3508
this should be fixed on master

**Comment by dgobaud** [comment]
@SomberNight got it any eta on a release? thanks
